js/background.js

```diff
@@ -112,8 +112,10 @@
     var code = 'document.body.style.fontFamily = "' + font + '";';
     code += 'document.body.style.fontSize = "' + fontSize + '";';
     code += 'var container = document.querySelector("pre");';
+    code += 'if (container) {';
```

Comment:
`container` was undefined sometimes during my testing. I should've removed that.